internal/util/symlink_walker.go

```diff
@@ -42,7 +42,12 @@ func (w *symlinkWalker) visit(fn AbsoluteWalkFunc) AbsoluteWalkFunc {
 			linkTarget, err := filepath.EvalSymlinks(path.String())
 			targetPath := NewPath(linkTarget, path.Fs())
 			if err != nil {
-				return fmt.Errorf("error following symlink %s: %w", path, err)
+				// There's software that creates symbolic links to files which do not exist,
```

Comment:
nit: Just above: Move the computation of `targetPath` after this error handling.

Reply:
Ah! nice catch, thanks!